twindle-cli/twitter/api/helpers/fetch.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+const nodeFetch = require("node-fetch").default;
+const { ApiErrors } = require("../../error");
+
+/**
+ * @param {string} url Twitter endpoint to send request to
+ * @param {string} token Bearer token. Should be provided from the .env file
+ */
+const fetch = async (url, token) => {
+  if (!token) throw new ApiErrors.TokenNotProvidedError();
+
+  const response = await nodeFetch(url, {
+    method: "GET",
+    headers: { Authorization: `Bearer ${token}` },
+    redirect: "follow",
+  });
+
+  // REVIEW WANTED: What should we do when its not 200? This below is just a workaround for now
+  if (response.status !== 200) throw new ApiErrors.NetworkRequestError();
```

Comment:
@Mira-Alf any thoughts based on the API?

Reply:
I think we should give more informative error messages. The user will be confused why the error if their Wifi is booming around

Reply:
@PuruVJ create an issue to discuss

Reply:
Sure